packages/x-svg-converter/src/__tests__/svg-to-vue.spec.ts

```diff
@@ -0,0 +1,112 @@
+import path from 'path';
+import fs from 'fs';
+import { exec } from 'child_process';
+import { svgToVue } from '../svg-to-vue';
+import svgStub from './svg-stub';
+
+// Mock to prevent the prettier from running in test env.
+jest.mock('child_process', () => {
+  const originalModule = jest.requireActual('child_process');
+
+  return {
+    __esModule: true,
+    ...originalModule,
+    exec: jest.fn(() => true)
+  };
+});
+
+describe('test SVG to Vue script', () => {
+  const sourcePath = './src/__tests__/svgs';
+
+  beforeEach(() => {
+    process.argv = ['param1', 'param2', sourcePath];
+
+    generateTestSVGs();
+  });
+
+  afterEach(() => {
+    removeVueTestFiles();
+  });
+
```

Comment:
Restore the original process arguments after the tests, just in case.

```suggestion

  afterAll(() => {
    process.argv = originalProcessArgv;
  });
  
```